docs/app/pages/components/components-sections/input-controls/radio-button/radio-button.component.html

```diff
@@ -52,8 +48,9 @@
 <hr>
 
 <p>
-    A radio button can be created by using the <code>ux-radio-button</code> component. The following attributes can be used
-    to configure the appearance and behavior of the component:
+    A radio button can be created by using the <code>ux-radio-button</code> component placed inside a
+    <code>uxRadioButtonGroup</code>. The following attributes can be used to configure the appearance and
+    behavior of the component:
 </p>
 
 <uxd-api-properties tableTitle="Inputs">
```

Comment:
Perhaps these titles should mention these are the inputs/outputs for the radio button

Reply:
Added that input/outputs for radio buttons